src/pages/[postcode]/places/place/place.page.tsx

```diff
@@ -13,10 +13,22 @@ import MaterialSearchInput from '@/components/control/MaterialSearchInput/Materi
 import '@/components/control/Details/Details';
 import useAnalytics from '@/lib/useAnalytics';
 import useFormValidation from '@/lib/useFormValidation';
-import { Location } from '@/types/locatorApi';
+import { Location, MaterialWithCategory } from '@/types/locatorApi';
 
 import { usePlaceLoaderData } from './place.loader';
 
+export function placeHasMaterialName(
+  search: string,
+  materials: MaterialWithCategory[] = [],
+): boolean {
+  return materials.some((material) => {
```

Comment:
This could be abstracted out to a library function, I see it repeated in `containerHasMaterialName`